### ISSUE TITLE
How do you create lookup fields?

### ISSUE THREAD

**Issue by hamayounCS** [mshanemc/shane-sfdx-plugins#72]
There seems to be no option to create a lookup field when creating a custom field

**Comment by mshanemc** [comment]
Yeah, haven’t done that type yet.  I’d take a PR otherwise you might be waiting until I really need it ;)

**Comment by mshanemc** [comment]
exists now, 4.31
